src/services/utilities.ts

```diff
@@ -438,6 +438,8 @@ namespace ts {
                 }
             case SyntaxKind.Identifier:
                 return isImportClause(node.parent) ? ScriptElementKind.alias : ScriptElementKind.unknown;
+            case SyntaxKind.ExportAssignment:

```

Comment:
I think this needs to handled similar to `AssignmentDeclarationKind.ModuleExports` so that export `default function foo()` shows the kind as function and not as const

Reply:
Yeah, makes sense - I've used `isFunctionLikeKind` to determine that and you can see some examples in the tests 👍

Reply:
Need to also handle ClassExpression?

Reply:
Instead of handling each case, I've switched to recurse in the `node.expression`

```ts
case SyntaxKind.ExportAssignment:
    const scriptKind = getNodeKind((node as ExportAssignment).expression);
    // If the expression didn't come back with something (like it does for an identifiers)
    return scriptKind === ScriptElementKind.unknown ? ScriptElementKind.constElement : scriptKind;
```

I'm open to this going a bit further and digging into the identifier and finding out out var/let/const, but I don't know how to pull out the declaration of a random identifier to check.